lib/results_screen.dart

```diff
@@ -0,0 +1,147 @@
+import 'package:distal_radius/image_handler.dart';
+import 'package:flutter/material.dart';
+import 'package:distal_radius/image_results_screen.dart';
+
+import 'dart:io';
+
+class ResultsScreen extends StatefulWidget {
+  const ResultsScreen({super.key});
+
+  static const String id = "results_screen";
+
+  @override
+  State<ResultsScreen> createState() => _ResultsScreenState();
+}
+
+// State class that handles logic in the screen
+// contains all functions relevant to business logic
+class _ResultsScreenState extends State<ResultsScreen> {
+  ImageHandler imageHandler = ImageHandler();
+
+  void cancelResults() {
+    print("Cancelled");
+  }
+
+  void toExport() {
+    print("Send to export");
+  }
+
+  void toImageResults(isFront) {
+    Navigator.push(
+        context,
+        MaterialPageRoute(
+            builder: (context) => ImageResultsScreen(
+                  isFrontImage: isFront,
+                )));
+  }
+
+  Widget getHeader() {
+    return Stack(
+      alignment: Alignment.center,
+      children: const [
+        Positioned(left: 10, child: BackButton()),
+        Align(
+            child: Text(
+          "Results",
+          textAlign: TextAlign.center,
+          textScaleFactor: 1.5,
+        )),
+      ],
+    );
+  }
+
+  Widget getImages() {
+    var frontImage = FileImage(File(imageHandler.frontImagePath!));
+    var sideImage = FileImage(File(imageHandler.sideImagePath!));
+
+    return Row(children: [
+      GestureDetector(
+          onTap: () => toImageResults(true),
+          child: Image(height: 200, width: 200, image: frontImage)),
+      GestureDetector(
+          onTap: () => toImageResults(false),
+          child: Image(height: 200, width: 200, image: sideImage))
+    ]);
+  }
```

Comment:
Same issue as before, the images overflow off the side of the screen (for some devices). Also, the images seem a bit small, may want to change them to be larger (like the other version in result-screen-text)